### ISSUE TITLE
CodeMirror 6 support

### ISSUE THREAD

**Issue by bmeurer** [codemirror/google-modes#343]
Are there any plans to migrate this to CodeMirror 6? Especially regarding the Angular templates and the Kotlin and Go support?

**Comment by marijnh** [comment]
No. This approach to parsing has been superseded by Lezer, which is preferable in just about every way.
